document-message/document-message.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/http;
+import ballerina/mime;
+
+type CsvRequest record {
+    string X_CRM_ORG;
+    string org;
+    string filename;
+    string path;
+};
+
+type ZohoResponse record {
+    string status;
+    string code;
+    string message;
+    record {|
+        string file_id;
+        string created_time;
+    |} details;
+};
+
+service /api on new http:Listener(8080) {
```

Comment:
Shall we keep the path as `/` here? or change it to a meaningful name

Reply:
renamed to `crm`